argo-cd-apps/base/member/infra-deployments/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - gitops
```

Comment:
I would prefer to point directly to gitops.yaml which could be in argo-cd-apps/base/member/infra-deployments - avoiding creation folder with another kustomization.yaml (which can be created later if needed.)

Reply:
The purpose of putting it and the other apps in their own folder is to allow picking which to use for each environment. Kustomize doesn't let you specify a specific yaml as a base, just Kustomize directories. At this point where everything changes rapidly, I would like to keep this flexibility.